hw/ip/uart/dv/uart_sim_cfg.hjson

```diff
@@ -2,7 +2,7 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 {
-  // Name of the sim cfg - typically same as the DUT under test.
```

Comment:
Also please update the template at `util/uvmdvgen/sim_cfg.hjson.tpl`

Reply:
Done, thanks for directing me to this file.